package_bag/routines.py

```diff
@@ -86,8 +86,8 @@ class RightsAssigner(object):
     """Send rights IDs to external service and receive JSON in return"""
 
     def run(self):
-        url = "rights service url"
-        apikey = "rights service apikey"
+        url = settings.RIGHTS_URL
+        apikey = settings.RIGHTS_KEY
```

Comment:
It's a little weird to me to pass these values into the `retrieve_rights` method. There's nothing wrong with it per-se, but you're always going to use those configured values (right?) so why not just grab them in `retrieve_rights` directly? 

If `retrieve_rights` were a more abstract client that would work against multiple apps/endpoints then you'd want to be able to configure it more flexibly. But it's pretty specific here.